reports/final/report.tex

```diff
@@ -1542,6 +1542,50 @@ \section{Stakeholder Needs} \label{sec:stakeholder-needs} % John / Tim
 % account with our implementation (which clearly shows time or FFT and both at the
 % same time)
 
+\subsubsection{Features and Quality}
+
+%% Ethan, if you could elaborate on why the kits were bad that would be great.
+%% John finds it difficult to comment since he did not use the kits
+%% Emphasis on the "quality" point
+
+%% and to double check, we did indeed have scroll to zoom functionality?
+
+In the online electrical engineering labs during the COVID pandemic, Professor
+Whitehall noticed issues with the kits provided to students. There was a general
+lack of quality and brittleness which caused students to require replacement
+kits. Additionally, the implementation of certain features was poor; the display
```

Comment:
"was poor" should be "were poor"

Reply:
8eca5b13e067e762d9c268a3869be6e17a714cce